packages/simplebar/src/simplebar.js

```diff
@@ -70,9 +70,6 @@ export default class SimpleBar {
 
     SimpleBar.getRtlHelpers = memoize(SimpleBar.getRtlHelpers);
 
-    // getContentElement is deprecated
-    this.getContentElement = this.getScrollElement;
-
```

Comment:
It's fine to remove this since we are bumping major version. We can drop the deprecated fallback.

Reply:
We could, but now that the content element is different from the scroll element (to make the ResizeObserver work), I figured it might be better to undeprecate it so there's still a supported way to add children to the content dynamically. Thoughts?

Reply:
Oh you're damn right. Would need to update the documentation.